packages/quick_actions/quick_actions/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Updates minimum Flutter version to 2.8.
 
+## 0.7.0
```

Comment:
@stuartmorgan Should this include the `NEXT` item since this is a version bump?

Reply:
Yes, but the larger problem is that this version bump is to the wrong package. The changes are to `quick_actions_android`, not `quick_actions`, so that's what needs CHANGELOG and version updates.

@TabooSun Please make sure the `publishable` CI check is passing when you update your PR; the failure messages there explain the problem.